tests/test-cmd/test.build.js

```diff
@@ -227,6 +227,15 @@ describe('build', () => {
       assert.equal(defaultFilter.wantFile('path/to/node_modules'), false);
     });
 
+    it('ignores the artifacts dir', () => withTempDir(
+      (tmpDir) => {
+        const filter = new FileFilter({
+          filePathsToIgnore: [path.join(tmpDir.path(), 'artifacts')],
```

Comment:
You don't need to create a temp directory. This argument doesn't have to be a real path. It could simply be `/example/path/to/web-ext-artifacts`